### DISCUSSION TITLE
Events created in Google web interfase lose reminders when updated in Thunderbird.

### DISCUSSION THREAD

**Discussion by Clod** [kewisch/gdata-provider#567] [Support]
If you create and event with more than one reminder (for example one alert and one email) in Google Calendar by means of web interfase and then you modify said event in Thunderbird (Windows 11 Thunderbird version 102.6.1 (64-bit) plugin version 102.0.3), only one of the reminders is kept and the other(s) are lost.
Did anyone notice the same behavior?
 
Thanks!

**Comment by kewisch** [comment]
Hi! Can you provide some error logs when you take this action? Here are the steps:
     * Open your application preferences (Tools or Edit Menu)
    * Select the `Advanced` icon at the top.
    * Click on the `General` tab, below the icons.
    * Select `Advanced Configuration` or `Config Editor`, whichever is available
    * Right Click on an empty space in the list of preferences
    * Select `New` → `Boolean`
    * Enter `calendar.debug.log` as the preference name
    * Pick `true` as the Boolean value.
    * Repeat the same for `calendar.debug.log.verbose`
